src/control/cmd/dmg/pretty/storage_query_test.go

```diff
@@ -400,10 +401,10 @@ T2   data
 
 Rank T1-Total T1-Usable T1-Usage T2-Total T2-Usable T2-Usage 
 ---- -------- --------- -------- -------- --------- -------- 
-0    2.0 TB   500 GB    75 %     2.0 TB   1.0 TB    50 %     
-1    2.0 TB   250 GB    87 %     2.0 TB   500 GB    75 %     
-2    1.0 TB   250 GB    75 %     1.0 TB   500 GB    50 %     
-3    1.0 TB   125 GB    87 %     1.0 TB   250 GB    75 %     
+0    2.0 TB   1.0 TB    50 %     2.0 TB   1.0 TB    50 %     
+1    2.0 TB   500 GB    75 %     2.0 TB   500 GB    75 %     
+2    1.0 TB   500 GB    50 %     1.0 TB   500 GB    50 %     
+3    1.0 TB   250 GB    75 %     1.0 TB   250 GB    75 %     
```

Comment:
Huh. Did one of your new changes affect this calculation? Or was the unit test failing?

Reply:
Yes this was a purposeful fix, after talking with Johann we decided that when displaying usable capacity for a META-only (no-DATA) role, "available" bytes should be returned because "usable" will be zero and only refers to capacity available for DATA (as per the adjustNvme functions performed during scan). For a META-only tier, usable == available.